R/plot_utils.R

```diff
@@ -185,3 +172,16 @@ getCxPlotDecorations <- function(decorations, color, width, x, y) {
     decorations <- list(line = append(decorations$line, list(line)))
     return(decorations)
 }
+
+cxSupportedLineFit <- function(linefit) {
+    if (!(linefit %in% c("lm", "loess"))) {
+        if (linefit == "glm") {
+            linefit <- "lm"
+        } else if (linefit == "gam") {
+            linefit <- "loess"
+        }
+        warning("Model type is not supported for canvasXpress charts and ", linefit," is being used")
```

Comment:
Would not it be useful to mention new **lineFit** type to the user?

Reply:
This function is not being used any more for plotDispersion. I have replaced that code. I have only retained it here as it is being used in a different function. Eventually this function will be removed.